apps/src/sites/code.org/pages/public/teacher-dashboard/sections.js

```diff
@@ -4,10 +4,14 @@ import { Provider } from 'react-redux';
 import { getStore, registerReducers } from '@cdo/apps/redux';
 import teacherSections, {
   setOAuthProvider,
-  asyncLoadSectionData,
+  asyncLoadSectionData
```

Comment:
supernit: any reason for removing the comma? I often add commas when I see them not here (tho it's fine not to have them).

my justification is that so that when you add/remove lines, you only see the affected lines (whereas right now if i wanted to add a line below this one, i would need to also change line 7.

not necessary to go through the effort of changing this back though.

Reply:
I think I added an import in one commit and then removed it which is why the comma is gone. But good point, noted for the future.